libiocage/lib/Release.py

```diff
@@ -36,8 +37,8 @@ def __init__(self, name=None,
         libiocage.lib.helpers.init_zfs(self, zfs)
         libiocage.lib.helpers.init_host(self, host)
 
-        if not libiocage.lib.helpers.validate_name(name):
-            raise NameError(f"Invalid 'name' for Release: '{name}'")
+        # if not libiocage.lib.helpers.validate_name(name):
+        #     raise NameError(f"Invalid 'name' for Release: '{name}'")
```

Comment:
Debug?

Reply:
Yes, had to comment the lines out until a solution to shorten the long HardenedBSD release names. Thanks for reminding me to re-enable those lines.